examples/example.py

```diff
@@ -32,6 +32,7 @@
             "geometry_type": ["Polygon", "MultiPolygon"],
             "crs": df.crs.to_wkt(pyproj.enums.WktVersion.WKT2_2019_SIMPLIFIED),
             "edges": "planar",
+            "orientation": "counterclockwise",
```

Comment:
This is wrong, as the example data is not actually using CCW rings. 

The current example file is converted from a Shapefile, which has by definition an opposite winding order.